tmt/steps/execute/internal.py

```diff
@@ -43,7 +48,7 @@
 SCRIPTS = (TMT_FILE_SUBMIT_SCRIPT, TMT_REBOOT_SCRIPT, TMT_REPORT_RESULT_SCRIPT)
 
 
-class ExecuteInternal(tmt.steps.execute.ExecutePlugin):
+class ExecuteInternal(tmt.steps.execute.ExecutePlugin):  # type: ignore[misc]
```

Comment:
Can we wait until #1327 gets merged? It is ready for review, so it shouldn't take too long. We could get rid of this `# type: ignore` thanks to that patch.

Reply:
+1, it should be merged very soon!

Reply:
ack. I see that it's merged so I'll rebase and remove this ignore.

Reply:
So now after I removed that, a bunch of new errors appear. 
```
tmt/steps/execute/internal.py:100: error: Argument 1 to "listify" has incompatible type "StepData"; expected "Union[Tuple[Any, ...], List[Any], str, Dict[Any, Any]]"
tmt/steps/execute/internal.py:148: error: Item "Common" of "Optional[CommonDerivedType]" has no attribute "plan"
tmt/steps/execute/internal.py:148: error: Item "None" of "Optional[CommonDerivedType]" has no attribute "plan"
tmt/steps/execute/internal.py:269: error: Signature of "go" incompatible with supertype "ExecutePlugin"
tmt/steps/execute/internal.py:269: note:      Superclass:
tmt/steps/execute/internal.py:269: note:          def go(self, *args: Any, **kwargs: Any) -> None
tmt/steps/execute/internal.py:269: note:      Subclass:
tmt/steps/execute/internal.py:269: note:          def go(self, guest: Any) -> None
tmt/steps/execute/internal.py:269: error: Signature of "go" incompatible with supertype "Plugin"
tmt/steps/execute/internal.py:269: note:          def go(self, *args: Any, **kwargs: Any) -> None
tmt/steps/execute/internal.py:269: note:          def go(self, guest: Any) -> None
tmt/steps/execute/internal.py:269: error: Signature of "go" incompatible with supertype "Phase"
tmt/steps/execute/internal.py:269: note:          def go(self, *args: Any, **kwargs: Any) -> None
tmt/steps/execute/internal.py:269: note:          def go(self, guest: Any) -> None
tmt/steps/execute/internal.py:342: error: "Step" has no attribute "_framework"
```

What is an acceptable way to resolve `Item "Common" of "Optional[CommonDerivedType]" has no attribute "plan"` ?

Reply:
@guy9050  ok, this is actually blocked at least on #1365, updating ... let's wait for that to be merged

Reply:
all mentioned MRs are now merged.